README.md

```diff
@@ -0,0 +1,2 @@
+# Aqovia.PactProducerVerifier
+A utility for verifying producer code against all consumers on the Pact Broker
```

Comment:
I would like to see more guidance on how to actually use this library, including simple examples or links to examples.